pages/events/index.vue

```diff
@@ -336,6 +359,12 @@ export default class EventsPage extends QiskitPage {
     }
   }
 
+  &__calendar{
+    margin-bottom: $spacing-06;
+    border: 1px solid #8d8d8d;
```

Comment:
I like the border you added, good idea. 

We should reuse one of our cool gray variables, try looking here at this [$border-color](https://github.com/Qiskit/qiskit.org/blob/main/assets/scss/helpers/_variables.scss#L36) one